wallets/rn_cli_wallet_068_5/src/components/HomeScreen/IndividualSession.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {View, Image, Text, StyleSheet} from 'react-native';
+
+interface IndividualSessionProps {
+  name: string | undefined;
+  icons: string;
+  url: string;
+}
+
+//ToDo: Change to TouchableOpacity and navigate to Session on next Sprint
+const IndividualSession = ({name, icons, url}: IndividualSessionProps) => {
+  return (
+    <View style={styles.sessionContainer}>
+      <View style={styles.flexRow}>
+        {icons ? (
+          <Image
+            source={{uri: icons.toString()}}
```

Comment:
Wouldn't this throw an exception if icons is empty?

Reply:
Returns null